src/types/events/base-events.ts

```diff
@@ -882,11 +885,14 @@ export interface UserChangeEvent {
       status_text: string;
       status_text_canonical: string;
       status_emoji: string;
+      status_emoji_display_info: [];
       status_expiration: number;
       avatar_hash: string;
+      huddle_state?: string;
+      huddle_state_expiration_ts?: number;
       first_name: string;
       last_name: string;
-      email: string;
+      email?: string;
```

Comment:
Didn't see this field either, even though the "is_email_confirmed" field WAS present and it said "true" 🤔